app/controllers/cypress/factories_controller.rb

```diff
@@ -7,26 +7,53 @@ module Cypress
   class FactoriesController < CypressController
     # Wrapper around FactoryBot.create to create a factory via a POST request.
```

Comment:
I think it is not quite precise here as it is not a factory that is created, rather the factory creates an instance of something.